storage/sql/config.go

```diff
@@ -242,6 +242,10 @@ func (s *MySQL) open(logger log.Logger) (*conn, error) {
 		if s.Host[0] != '/' {
 			cfg.Net = "tcp"
 			cfg.Addr = s.Host
+
+			if s.Port != 0 {
+				cfg.Addr = fmt.Sprintf("%s:%d", s.Host, s.Port)
```

Comment:
Maybe it is better to use `net.JoinHostPort` to cover the case with IPv6 addresses? However, it is required port to be a string.

Reply:
Makes sense, thanks!